external/vcpkg_custom_data/toolchains/linux_32.cmake

```diff
@@ -0,0 +1,572 @@
+# Toolchain file for building 32-bit Linux libraries
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -m32")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m32")
+set(CMAKE_LIBRARY_PATH "/usr/lib/i386-linux-gnu")
+
+# Mark variables as used so cmake doesn't complain about them
+mark_as_advanced(CMAKE_TOOLCHAIN_FILE)
+
+# VCPKG toolchain options.
+option(VCPKG_VERBOSE "Enables messages from the VCPKG toolchain for debugging purposes." OFF)
+mark_as_advanced(VCPKG_VERBOSE)
+
+include(CMakeDependentOption)
+
+function(_vcpkg_get_directory_name_of_file_above OUT DIRECTORY FILENAME)
+    set(_vcpkg_get_dir_candidate ${DIRECTORY})
+    while(IS_DIRECTORY ${_vcpkg_get_dir_candidate} AND NOT DEFINED _vcpkg_get_dir_out)
+        if(EXISTS ${_vcpkg_get_dir_candidate}/${FILENAME})
+            set(_vcpkg_get_dir_out ${_vcpkg_get_dir_candidate})
+        else()
+            get_filename_component(_vcpkg_get_dir_candidate_tmp ${_vcpkg_get_dir_candidate} DIRECTORY)
+            if(_vcpkg_get_dir_candidate STREQUAL _vcpkg_get_dir_candidate_tmp) # we've reached the root
+                set(_vcpkg_get_dir_out "${OUT}-NOTFOUND")
+            else()
+                set(_vcpkg_get_dir_candidate ${_vcpkg_get_dir_candidate_tmp})
+            endif()
+        endif()
+    endwhile()
+
+    set(${OUT} ${_vcpkg_get_dir_out} CACHE INTERNAL "_vcpkg_get_directory_name_of_file_above: ${OUT}")
+endfunction()
+
+if(NOT DEFINED VCPKG_MANIFEST_DIR)
+    if(EXISTS "${CMAKE_SOURCE_DIR}/vcpkg.json")
+        set(_VCPKG_MANIFEST_DIR "${CMAKE_SOURCE_DIR}")
+    endif()
+else()
+    set(_VCPKG_MANIFEST_DIR ${VCPKG_MANIFEST_DIR})
+endif()
+if(NOT DEFINED VCPKG_MANIFEST_MODE)
+    if(_VCPKG_MANIFEST_DIR)
+        set(VCPKG_MANIFEST_MODE ON)
+    else()
+        set(VCPKG_MANIFEST_MODE OFF)
+    endif()
+elseif(VCPKG_MANIFEST_MODE AND NOT _VCPKG_MANIFEST_DIR)
+    message(FATAL_ERROR
+        "vcpkg manifest mode was enabled, but we couldn't find a manifest file (vcpkg.json) "
+        "in any directories above ${CMAKE_CURRENT_SOURCE_DIR}. Please add a manifest, or "
+        "disable manifests by turning off VCPKG_MANIFEST_MODE.")
+endif()
+
+if(NOT DEFINED _INTERNAL_CHECK_VCPKG_MANIFEST_MODE)
+    set(_INTERNAL_CHECK_VCPKG_MANIFEST_MODE "${VCPKG_MANIFEST_MODE}"
+        CACHE INTERNAL "Making sure VCPKG_MANIFEST_MODE doesn't change")
+endif()
+
+if(NOT VCPKG_MANIFEST_MODE STREQUAL _INTERNAL_CHECK_VCPKG_MANIFEST_MODE)
+    message(FATAL_ERROR [[
+vcpkg manifest mode was enabled for a build directory where it was initially disabled.
+This is not supported. Please delete the build directory and reconfigure.
```

Comment:
Any direction that we can point the user to reconfigure?  Is it re-run ./configure ?